packages/perseus/src/widgets/interactive-graphs/interactive-graph-question-builder.ts

```diff
@@ -695,7 +695,7 @@ class PointsGraphConfig implements InteractiveFigureConfig {
         },
     ) {
         this.numPoints = numPoints;
-        this.coords = options?.coords ?? [[0, 0]];
+        this.coords = options?.coords;
```

Comment:
Question: I don't think this affects this PR, but you mentioned in the summary that you changed this so it "calculates the correct coords itself." How does it do that?

Reply:
If I don't pass in any coords, then the graph automatically initializes with the default starting coords in `initialize-graph-state.ts` when it renders. If I hard-code it here, it doesn't initialize to the right one based on the number of points. I could also call the initializer function here, but that would be redundant.